src/components/widgets/Comments/Comment/Comment.js

```diff
@@ -53,6 +55,13 @@ const Comment = ({
       </div>
       <img className="direct-chat-img" src={user.avatarUrl} alt={user.name} />
       <div className="direct-chat-text">
+        {isFromCurrentUser &&
+          <DeleteIcon
+            gapLeft
+            className="pull-right"
+            style={{ cursor: 'pointer' }}
```

Comment:
This will not be necessary, I have just completed modification of an Icon (in another PR), which automatically sets pointer cursor to icons which have onClick or onDoubleClick set.